packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -133,23 +134,63 @@ function getLoadDocumentsOptions(filename: string) {
   return loadTypedefsConfig
 }
 
+function getPrismaClient(hasGenerated = false): {
+  ModelName: Record<string, string> | undefined
+} {
+  const localPrisma = require('@prisma/client')
+
+  if (!localPrisma.ModelName) {
+    if (hasGenerated) {
+      return { ModelName: undefined }
+    } else {
+      if (process.env.NODE_ENV === 'test') {
```

Comment:
Isn't it better to mock the prismsa client in Redwood tests than have this if block here?

Reply:
(see #6228)

Reply:
Totally! Thanks for the suggestion 👍